firewall/interception/windowskext/kext.go

```diff
@@ -142,9 +91,14 @@ func Stop() error {
 	}
 	ready.UnSet()
 
-	rc, _, lastErr := kext.stop.Call()
-	if rc != windows.NO_ERROR {
-		return formatErr(lastErr, rc)
+	err := closeDriver(kextHandle)
+	if err != nil {
+		log.Errorf("winkext: failed to close the handle: %s", err)
+	}
+
+	_, err = exec.Command("sc", "stop", driverName).Output() // This is a question of taste, but it is a robust and solid solution
+	if err != nil {
+		log.Errorf("winkext: failed to stop the service: %q", err)
 	}
```

Comment:
Why not use the service manager here?
What makes it robust and solid?
I find it weird to use one way to start and another to stop it.